app/src/main/java/org/oppia/app/topic/TopicFragmentPresenter.kt

```diff
@@ -53,6 +53,10 @@ class TopicFragmentPresenter @Inject constructor(
     return binding.root
   }
 
+  fun setCurrentTab(tabNumber: Int) {
```

Comment:
Can we use the enum, instead?  I think that would be a lot cleaner--let's avoid using integers when we have a nice type-safe enum to use instead. It has the added benefit of making it clear at the callsite which tab is being selected (I personally don't know which tab is at position 1 offhand without thinking about it or loading the app/mocks to verify).

Reply:
used 
` fun setCurrentTab(tab: TopicTab) {
    viewPager.setCurrentItem(tab.ordinal, true)
  }`